test/integration/compile-swagger-test.coffee

```diff
@@ -97,25 +97,44 @@ describe('compile() · Swagger', ->
   )
 
   describe('with multiple responses', ->
-    # Multiple responses is the closest we can get with Swagger to something
-    # like transaction examples ¯\_(ツ)_/¯
-
+    transactions = undefined
+    filename = 'apiDescription.json'
     detectTransactionExamples = sinon.spy(require('../../src/detect-transaction-examples'))
-    compilationResult = undefined
-    transaction = undefined
+
+    expectedStatusCodes = [200, 400, 500]
```

Comment:
I would change the naming in way it supports the fact you are iterating over it and evaluating all of them.